### PR TITLE
Added private vesting termination option.

### PR DESCRIPTION
**Pull request by AngelBlock**
- [x] minor layout fix
- [x] private vesting termination

### CODE REVIEW

**Comment by ilblackdragon** [comment]
Please remove .idea folder and add it to .gitignore

**Comment by AngelBlock** [comment]
> Please remove .idea folder and add it to .gitignore

Done, please review

File: actions.js [hunk 1]
```diff
@@ -1,228 +1,281 @@
...
+      {
+        vesting_schedule:
+          {
+            start_timestamp: lockupVestingStartDate.toString(),
+            cliff_timestamp: lockupVestingCliffDate.toString(),
+            end_timestamp: lockupVestingEndDate.toString()
+          },
+        salt
+      }
+  }
```
Comment (ilblackdragon) [inline actions.js:207]: it would be good to check here that the hash of this vesting actually matches hash in the contract. Otherwise one can submit a request only to find it's actually wrong after it got confirmed.

File: actions.js [hunk 2]
```diff
@@ -1,228 +1,281 @@
...
+  //console.log(args);
+  //args = Buffer.from(JSON.stringify(args)).toString('base64');
+  //console.log(args);
```
Comment (ilblackdragon) [inline actions.js:210]: remove this?

File: actions.js [hunk 3]
```diff
@@ -1,228 +1,281 @@
...
-    try {
-        duration = parseInt(duration);
-    } catch (error) {
-        alert(`Failed to parse duration ${duration}`);
-        return;
-    }
-    // Days to nano seconds.
-    duration = duration * 60 * 60 * 24 * 1000 * 1000 * 1000;
-    await deployLockup(contract, accountId, amount, duration, allowStaking);
+  let accountId = document.querySelector('#create-lockup-account-id').value;
```
Comment (ilblackdragon) [inline actions.js:230]: why is everything reformatted, actually?
may be reformat if really needed in separate PR? and add a js linter than to format automatically

**Review by ilblackdragon — APPROVED** [review]
Approve but add the check and next time break up formatting into separate PR plz

File: actions.js (hunk 3 above)
Comment (AngelBlock) [reply actions.js:230]: Apologies, reformatting was down automatically by the linter, I'll switch it off next time to a manual mode.

File: actions.js (hunk 1 above)
Comment (AngelBlock) [reply actions.js:207]: I suggested it but was decided to skip checks as it is a temporary solution before functionally moved to the lockup UI, and properly addressed, but thanks  for pointing out anyway.
